packages/deploy-utils/src/utils/git/tag.test.ts

```diff
@@ -0,0 +1,14 @@
+import { getLatestTag } from ".";
+import { DeployableApp } from "../types";
+
+describe("getLatestTag", () => {
+  const currentlyDeployedTagRegex = (app: DeployableApp) =>
+    new RegExp(`${app}/v\\d+.\\d+.\\d+`);
```

Comment:
It's not possible to assert an exact string value since we aren't mocking the value returned by `execSync` in `getLatestTag`. IMO that would defeat the usefulness of this test since we want to assert that the git tags are filtered correctly. I did add a test to verify that the regex matcher works against a known-valid string!